### PR TITLE
Adds single pdf scraper with bonus repo wide format standardization 

### PR DESCRIPTION
**Pull request by CaptainStabs**

### CODE REVIEW

**Review by EricTurner3 — DISMISSED** [review]
Approved:

- I like the PEP8 formatting, Stabs said he used this: https://github.com/psf/black
- Also added antiocha scraper that scrapes PDF files
- There are no terms and conditions on the website so this scraper fits legal requirements

**Review by CaptainStabs — COMMENTED** [review]
Shouldn't break anything 🤞

File: USA/CA/Brea/brea_annual_scraper.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import sys
+from pathlib import Path
+p = Path(__file__).resolve().parents[3]
+sys.path.insert(1, str(p) + "/common")
+from base_scrapers.list_pdf_scrapers.single_pdf_scraper import single_pdf_scraper
+
+url_2 = "https://www.ci.brea.ca.us/DocumentCenter/View/116/CrimeStats_Brea"
```
Comment (EricTurner3) [inline USA/CA/Brea/brea_annual_scraper.py:7]: Full Policy found here: https://www.ci.brea.ca.us/DocumentCenter/View/1329/Website-Content-User-Policy

File: USA/CA/COL_LosRiosComm/hash.txt [hunk 2]
```diff
@@ -1 +1 @@
-470aaeb961c2063fd84528efebcaa98a
\ No newline at end of file
+50bc9a673d0d3ca790c1e409e3854821
```
Comment (EricTurner3) [inline USA/CA/COL_LosRiosComm/hash.txt:1]: hash is used to verify the data from the last time the script ran is different

File: USA/CA/antiocha/antiocha_scraper.py [hunk 3]
```diff
@@ -9,7 +9,7 @@
 
 p = Path(__file__).resolve().parents[3]
 sys.path.insert(1, str(p) + "/common")
-from base_scrapers.get_files import get_files
+from common.get_files import get_files
 
 webpage = "https://www.antiochca.gov/police/crime-statistics/crime-statistics/"
```
Comment (EricTurner3) [inline USA/CA/antiocha/antiocha_scraper.py:14]: searched the site, did not find any terms of service that we would breach

**Review by EricTurner3 — COMMENTED** [review]
This massive PR entails the following:

- PEP8 format standarization of all files, using the black formatter
- Modifies import statement of `list_pdf_scrapers` module to `base_scrapers`
- new single_pdf_scraper base script & configs
- Modifies import statement of `bs_scrapers` to `common.file_downloaders`
- Uses a variation of single_pdf_scraper to USA/CA/Brea
- new scraper for /USA/CA/antiocha
- modifies import statement of `base_scrapers` to `common` for get_files
- adds a date mechanism to `crimegraphics` scraper to prevent overwrite
- adds new `list_pdf_v3` scraper in `common`
- adds new `single_pdf_scraper` in `common`
- adds `__init__` files for the refactored module directories in `common`
- adds new `downloaders` module in `common\file_downloaders`

Everything looks good, however I am requesting more review on the Legality of /USA/CA/Brea scraper from @josh-chamberlain before approving. Terms of service for their site can be found here: https://www.ci.brea.ca.us/DocumentCenter/View/1329/Website-Content-User-Policy

**Review by EricTurner3 — CHANGES_REQUESTED** [review]
* Remove the scraper at /USA/CA/Brea pending legal review. It can be added back in another PR.

**Review by EricTurner3 — APPROVED** [review]
- I see several readme's from commits within the past hour
- removed /USA/CA/Brea

**Pull request merged**